### DISCUSSION TITLE
Loss Weights in stokes.py for hPINN

### DISCUSSION THREAD

**Discussion by ppandit95** [lululxvi/deepxde#835] [Q&A] [answered]
Dear Dr Lu

I am trying to implement hPINN and in the process I reached a point where I am kind of unclear about the meaning of notations used in stokes.py in hPINN repository specifically in loss_weights as
`loss_weights = [mu_PDE / 3] * 3 + [0] * 3 + [mu_V, 0] + [1]`

In here,the third square bracket which involves mu_V as well as 0 concerns me.So it will be truly helpful if you could comment on this .

Many Thanks
Pushkar

**Comment by lululxvi** [comment, marked as answer]
It simply means the weights are mu_V and 0.

**Comment by ppandit95** [comment]
Thanks @lululxvi for elaborating on that
